runtime/src/main/kotlin/kotlin/native/simd.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package kotlin.native
+
+import kotlin.native.internal.*
+import kotlinx.cinterop.*
```

Comment:
Is this import used?
Importing `kotlinx.cinterop.*` to `stdlib` is kinda wrong.